include/deal2lkit/parsed_quadrature.h

```diff
@@ -0,0 +1,91 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2015 by the deal2lkit authors
+//
+//    This file is part of the deal2lkit library.
+//
+//    The deal2lkit library is free software; you can use it, redistribute
+//    it, and/or modify it under the terms of the GNU Lesser General
+//    Public License as published by the Free Software Foundation; either
+//    version 2.1 of the License, or (at your option) any later version.
+//    The full text of the license can be found in the file LICENSE at
+//    the top level of the deal2lkit distribution.
+//
+//-----------------------------------------------------------
+
+#ifndef _d2k_parsed_quadrature
+#define _d2k_parsed_quadrature
+
+#include <deal.II/base/quadrature_selector.h>
+
+#include <deal2lkit/parameter_acceptor.h>
+
+using namespace dealii;
+
+
+D2K_NAMESPACE_OPEN
+
+/**
+ * A deal2lkit wrapper for dealii::QuadratureSelector.
+ * The template integegers specify the dimension of the quadrature.
+ */
+template<int dim>
+class ParsedQuadrature : public ParameterAcceptor
```

Comment:
Sarebbe molto più elegante, visto che è possibile copiare quadratures, se ParsedQuadrature fosse derivata oltre che da ParameterAcceptor, anche da `Quadrature<dim>`. In questo modo e' una quadrature vuota, fino a che non viene inizializzata, e poi diventa la quadrature selezionata...